src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/infrastructure/config/JacksonConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.iahomeofficeintegrationapi.infrastructure.config;
+
+import static com.fasterxml.jackson.databind.DeserializationFeature.READ_ENUMS_USING_TO_STRING;
+import static com.fasterxml.jackson.databind.DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_USING_DEFAULT_VALUE;
+import static com.fasterxml.jackson.databind.SerializationFeature.WRITE_ENUMS_USING_TO_STRING;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
+
+@Configuration
+public class JacksonConfiguration {
+
+    @Bean
+    @Primary
+    public Jackson2ObjectMapperBuilder jackson2ObjectMapperBuilder() {
+        return new Jackson2ObjectMapperBuilder()
```

Comment:
We want to ignore unknown properties.  If the home office add anything new our code should not break.  This way we do not need to map every field and only the ones we care about.

Reply:
Added configuration to ignore unknown fields